### ISSUE TITLE
Install Fails on DigitalOcean due to small kubecost-cost-analyzer PVC size

### ISSUE THREAD

**Issue by blairdrummond** [kubecost/cost-analyzer-helm-chart#507]
**Describe the bug**

Digital Ocean `do-block-storage` has a minimum storage size of 1gb, which causes [this line](https://github.com/kubecost/cost-analyzer-helm-chart/blob/d3bd31ac65e3f14546b4ffb622e3e125985f0386/cost-analyzer/values.yaml#L134) to prevent the PVC for kubecost-cost-analyzer from being created.

**The fix is as simple as running with**

```
helm install kubecost kubecost/cost-analyzer 
    --namespace kubecost 
    --set persistentVolume.size=1Gi            # all that's required
    --set kubecostToken="$REDACTED"

# a few minutes and a port-forward later, everything is working
```

**To Reproduce**
1. Have a default digital ocean cluster
2. Try to follow the default installation guide
3. cost-analyzer pod is stuck in pending due to un-provisionable pvc

**Expected behavior**
Pods and PVCs should get provisioned.

**Some logs**

```
$ k describe pod kubecost-cost-analyzer-65dd944b7-9gmkb -n kubecost

...

Events:
  Type     Reason             Age                 From                Message
  ----     ------             ----                ----                -------
  Warning  FailedScheduling   <unknown>           default-scheduler   error while running "VolumeBinding" filter plugin for pod "kubecost-cost-analyzer-65dd944b7-9gmkb": pod has unbound immediate PersistentVolumeClaims
  Warning  FailedScheduling   <unknown>           default-scheduler   error while running "VolumeBinding" filter plugin for pod "kubecost-cost-analyzer-65dd944b7-9gmkb": pod has unbound immediate PersistentVolumeClaims
  Normal   NotTriggerScaleUp  97s (x21 over 21m)  cluster-autoscaler  pod didn't trigger scale-up (it wouldn't fit if a new node is added):
```

*Unfortunately I closed the terminal that had the pvc description, but it was stuck because the lower-limit on do-block-storage is 1Gi*

**Comment by dwbrown2** [comment]
Thanks for reporting @blairdrummond! Sounds like everything else looked good after increasing this request to 1Gb?

**Comment by dwbrown2** [comment]
Re-opening, so that we can discuss options to support this by default + ETL to disk.

**Comment by micya** [comment]
Noticing a similar issue when installing on AKS:

```
$ kubectl describe pvc -n kubecost kubecost-cost-analyzer
Name:          kubecost-cost-analyzer
Namespace:     kubecost
StorageClass:  default
Status:        Pending
Volume:
Labels:        app=cost-analyzer
               app.kubernetes.io/instance=kubecost
               app.kubernetes.io/managed-by=Helm
               app.kubernetes.io/name=cost-analyzer
               helm.sh/chart=cost-analyzer-1.68.0
Annotations:   meta.helm.sh/release-name: kubecost
               meta.helm.sh/release-namespace: kubecost
               volume.beta.kubernetes.io/storage-provisioner: kubernetes.io/azure-disk
Finalizers:    [kubernetes.io/pvc-protection]
Capacity:
Access Modes:
VolumeMode:    Filesystem
Mounted By:    kubecost-cost-analyzer-79bcdc89dd-znpbr
Events:
  Type     Reason              Age                  From                         Message
  ----     ------              ----                 ----                         -------
  Warning  ProvisioningFailed  112s (x46 over 88m)  persistentvolume-controller  Failed to provision volume with StorageClass "default": quantity 214748364800m is too great, overflows int64
```

Setting the persistentVolume.size worked.

**Comment by dwbrown2** [comment]
thanks for sharing, @micya. can you share what version of aks you are running and how you installed kubecost?

cc @AjayTripathy

**Comment by micya** [comment]
Multiple versions of AKS: 1.17.5, 1.19.3.

Installation with Helm 3:

```
kubectl create namespace kubecost
helm repo add kubecost https://kubecost.github.io/cost-analyzer/
helm install kubecost kubecost/cost-analyzer --namespace kubecost --set kubecostToken="<redacted>"
```

Fixed with below change
```
helm install kubecost kubecost/cost-analyzer --namespace kubecost --set kubecostToken="<redacted>" --set persistentVolume.size=1Gi
```

**Comment by AjayTripathy** [comment]
Thanks for this report! This seems like it doesn't work on enough providers by default that we should consider simply bumping the default to 1GB here.

**Comment by dwbrown2** [comment]
If we're changing the default, should we increase to 32gb to host ETL as well?

**Comment by amccool** [comment]
I required the `--set persistentVolume.size=1Gi`  on helm3 AKS 1.19.3 as well

**Comment by dwbrown2** [comment]
Thanks again for the reports, @amccool and @blairdrummond. We plan to address this week by updating to 32gb now that our ETL cache is written to disk.

**Comment by dwbrown2** [comment]
Fixed in [v1.72.0](https://github.com/kubecost/cost-analyzer-helm-chart/releases/tag/v1.72.0)! Please let me know if you have any questions or additional suggestions here. Thank you again!
